Automattic/jetpack
# Instant Search: Implement cancellable API requests

Part of #13366.

#### Changes proposed in this Pull Request:
* Replaces `unfetch` ponyfill with `axios`, which supports cancellable requests using a cancel token API.
* When a new search request is fired, old search requests will be cancelled.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Using a Jetpack site, test these changes by searching your site. Confirm that XHR network requests are being canceled by newer requests in the browser network console. @bluefuton note: it's easiest to see this in Chrome.
* Using a private Atomic site, repeat the above test step.

#### Proposed changelog entry for your changes:
* None.
